bindings/c/src/lib.rs

```diff
@@ -31,6 +31,11 @@ unsafe fn set_err_msg(msg: String, output: *mut *const std::ffi::c_char) {
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn libsql_enable_tracing() {
+    tracing_subscriber::fmt::init();
```

Comment:
I wonder if we wanna do `let _ = tracing_subscriber::fmt::try_init()` so that repeated calls of this in the same process won't panic which I think would be quite hard to recover from in C code?

Reply:
Sure! I guess we can also propagate return code which can be useful if for some reason tracing can't be enabled:
```
if tracing_subscriber::fmt::try_init().is_ok() {
    1
} else {
    0
}
```